lms/templates/dashboard/course.underscore

```diff
@@ -0,0 +1,643 @@
+                                                       data-refund-info="<%- gettext('You will be refunded the amount you paid.') %>">
+                                                        <%- gettext('Unenroll') %>
+                                                    </a>
+                                                <% }
+                                            } else {
+                                                if (!model.is_course_blocked) { %>
+                                                    <a href="#unenroll-modal" class="action action-unenroll"
+                                                       rel="leanModal" data-course-id="<%- model.course_key %>"
+                                                       data-course-number="<%- model.course_number %>"
+                                                       data-course-name="<%- model.course_name %>"
+                                                       data-dashboard-index="<%- dashboard_index %>"
+                                                       data-cert-name-long="<%- settings.cert_name_long %>"
+                                                       data-track-info="<%- gettext('Are you sure you want to unenroll from the verified %(cert_name_long)s track of %(course_name)s (%(course_number)s)?') %>"
+                                                       data-refund-info="<%- gettext('The refund deadline for this course has passed, so you will not receive a refund.') %>">
+                                                        <%- gettext('Unenroll') %>
+                                                    </a>
+                                                <% } else { %>
+                                                    <a class="action action-unenroll is-disabled"
+                                                       data-course-id="<%- model.course_key %>"
+                                                       data-course-number="<%- model.course_number %>"
+                                                       data-course-name="<%- model.course_name %>"
+                                                       data-dashboard-index="<%- dashboard_index %>"
+                                                       data-cert-name-long="<%- settings.cert_name_long %>"
+                                                       data-track-info="<%- gettext('Are you sure you want to unenroll from the verified %(cert_name_long)s track of %(course_name)s (%(course_number)s)?') %>"
+                                                       data-refund-info="<%- gettext('The refund deadline for this course has passed, so you will not receive a refund.') %>">
+                                                        <%- gettext('Unenroll') %>
+                                                    </a>
+                                                <% }
```

Comment:
@afeef I'm not sure I understand how the template has access to values that the view does not? Isn't the view just passing data to the template? I was suggesting that the view update the data and do the calculations so the template is less complex. The view is really where most of the calculations / data-manipulation should take place.